components/inputs/input-date-range.js

```diff
@@ -129,16 +141,44 @@ class InputDateRange extends RtlMixin(LocalizeCoreElement(LitElement)) {
 		if (input) input.focus();
 	}
 
+	async validate() {
+		const errors = await super.validate();
+		if (errors.length !== 0) {
+			return errors;
+		}
+		return Promise.all([
+			await this.shadowRoot.querySelector('.d2l-input-date-range-start').validate(),
+			await this.shadowRoot.querySelector('.d2l-input-date-range-end').validate()]
+		).then((res) => {
+			return res.reduce((acc, errors) => [...acc, ...errors], []);
+		});
```

Comment:
This will likely be okay but we'll have to see how the tooltips end up working. We may only want to validate one or the other to prevent tooltip collisions.